src/renderer/core/layout/store/layoutStore.ts

```diff
@@ -138,6 +138,8 @@ class LayoutStoreImpl implements LayoutStore {
 
   // Vue dragging state for selection toolbox (public ref for direct mutation)
   public isDraggingVueNodes = ref(false)
+  // Vue resizing state to prevent drag from activating during resize
+  public isResizingVueNodes = ref(false)
```

Comment:
Maybe?

```suggestion
public readonly isDraggingVueNodes = ref(false)
  // Vue resizing state to prevent drag from activating during resize
  public readonly isResizingVueNodes = ref(false)
```